### PR TITLE
Fixing the CI errors

### PR DESCRIPTION
**Pull request by elichai**
- Updated to the new yml format, the old one failed mounting `isgx` on the new drone version.

- Locked `cbindgen` to `0.8.4` (https://github.com/eqrion/cbindgen/issues/335)
- Locked `openssl-sys` to `0.9.44` (need to update `rustc` https://github.com/sfackler/rust-openssl/pull/1103 )
- Locked `rust-rocksdb` to `0.12.1` (https://github.com/rust-rocksdb/rust-rocksdb/issues/304)

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,55 +1,65 @@
-pipeline:
-  core:
-    group: tests
+kind: pipeline
+name: default
+steps:
```
Comment (moriaab) [inline .drone.yml:3]: As far as I understand, "steps" will lead to each step to be executed sequentially. If so, why is the advantage over "group"?

Comment (elichai) [reply .drone.yml:3]: Yes, so everything I said about the yml today is deprecated lol
This is a new format, and yes steps are usually sequencial but adding the `depemds_on=[clone] ` makes them parallel.

Comment (moriaab) [reply .drone.yml:3]: depends_on=[clone] was my next question. This new .yml format is so non-intuitive :)))

Comment (elichai) [reply .drone.yml:3]: Yeah, they made it worse.

File: .drone.yml [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
       - . /root/.cargo/env
       - cd enigma-crypto
       - RUSTFLAGS=-Awarnings cargo test
       - RUSTFLAGS=-Awarnings cargo build --no-default-features --features=sgx
 
-  tools_m:
-    group: tests
-    image: enigmampc/enigma-core:0.0.5
+  - name: tools_m
+    image: enigmampc/enigma-core:0.0.5-rocksdb-5.17.2
```
Comment (moriaab) [inline .drone.yml:54]: Did you catch the wrong docker image version by accident?

**Review by moriaab — APPROVED** [review]
Well done! It was not trivial as drone changed the configuration syntax.
@AvishaiW I suggest you to go through the changes in drone.yml

**Pull request merged**